sparseqr/sparseqr.py

```diff
@@ -200,7 +200,7 @@ def cholmod_free_dense( A ):
 
 ## Solvers
 
-def qr( A, tolerance = None ):
+def qr( A, tolerance = None, economy = False ):
```

Comment:
Can you please make the default value `None` and then add the line `if economy is None: economy = False`? It makes it easier to wrap or extend functions that way. https://graysonkoonce.com/always-use-none-for-default-args-in-python/